### PR TITLE
Run three years

### LINKED ISSUE #213: Compare weather years
We have talked about whether we want to run all simulations over several (~10) years or whether it is enough to just run the simualtions for one year. 

I took a look at 
1) 7 years (2010 - 2017) for Berlin Germany with cpv (I will also run simulations for si and add them here later on). 

2) 7 years (2010 - 2017) for Madrid, Spain with cpv.

Here's the output. On the top you see only the weather data (irradiance and temperature) on the bottom I plotted some simulation outputs KPI. 

BERLIN, GERMANY
weather:
plot_compare_weatherdata_52 52437_13 41053

simulation results / KPI

plot_scalars_Scenario_W_G_cpv_Scenario_W_G_si_year

MADRID, SPAIN
weather:
plot_compare_weatherdata_40 416775_-3 70379

simulation results / KPI
plot_scalars_Scenario_W_S_cpv_Scenario_W_S_si_year

### PR DESCRIPTION
**Pull request by Piranias**
Fix #213 

**Changes proposed in this pull request**:
- this PR builds up on PR#217. This is why PR#217 is deprecated.
- By entering a list of years, the loop is wrapped by the "year" loop
- the loop_kpi function is adapted for the new output. It now plots various dots for the number of years that we simulate (for our simulation that would be 3)

The following steps were realized, as well (if applies):
- [x] Use in-line comments to explain your code
- [x] Write docstrings to your code
- [ ] For new functionalities: Explain in readthedocs
- [x] Write test(s) for your new patch of code
- :x: Update the CHANGELOG.md (let's start this after the first release)
- [x] Apply black (`black . --exclude docs/`)
- [x] Check if full simulation tests pass locally (`EXECUTE_TESTS_ON=master pytest`)

*Please mark above checkboxes as following:*
- [ ] Open
- [x] Done

:x: Check not applicable to this PR

Here's an example output for a loop over lifetime. It's just to check how the plot would look:
plot_scalars_Scenario_A1_lifetime

### CODE REVIEW

**Comment by Piranias** [comment]
About the plot: please write me your comments if you would change something about the plot. This is how I would then analyze our results and compare subscenarios (e.g dieffernt locations, technologies (as in the example) and so on.

**Review by SabineHaas — APPROVED** [review]
Nice feature :) 

I only did a quick review, without letting the code run. To me it looks fine! 
@Piranias I think all the files in `tests/data/outputs` are not needed online, right? I would upload none of the outputs except these are **expected** outputs that you use for testing.

@MaGering if you want to you could check whether the functionality works well for you. If you don't find the time it's no problem: as we have said earlier, the code will be tested during simulations, again.

**Comment by Piranias** [comment]
> Nice feature :)
> 
> I only did a quick review, without letting the code run. To me it looks fine!
> @Piranias I think all the files in `tests/data/outputs` are not needed online, right? I would upload none of the outputs except these are **expected** outputs that you use for testing.
> 
> @MaGering if you want to you could check whether the functionality works well for you. If you don't find the time it's no problem: as we have said earlier, the code will be tested during simulations, again.

the left over files are needed for the test_outputs.py functons. I deleted all files that are not needed.

**Comment by MaGering** [comment]
> @MaGering if you want to you could check whether the functionality works well for you. If you don't find the time it's no problem: as we have said earlier, the code will be tested during simulations, again.

Yes, I will test the functions superficially for now. :)

File: pvcompare/outputs.py [hunk 1]
```diff
@@ -135,128 +139,131 @@ def loop_pvcompare(
...
+                    plot=False,
+                    pv_setup=pv_setup,
+                    loop_output_directory=loop_output_directory,
+                    step=temp_high,
+                    loop_type=loop_type,
+                )
+                temp_high = temp_high + loop_dict["step"]
 
 
 def single_loop_pvcompare(
```
Comment (MaGering) [inline pvcompare/outputs.py:269]: Can it be that `scenario_name` is missing as input parameter of `single_loop_pvcompare(...)` function? Later on it is needed here:

 ```
main.apply_mvs(
        scenario_name,
        user_inputs_mvs_directory=user_inputs_mvs_directory,
        mvs_output_directory=mvs_output_directory,
        outputs_directory=outputs_directory,
    )
```

but hasn't been defined before or has it?

You can reconstruct this running `scenarios.run_scenario_a7()` in `run_greco_scenarios.py` on your branch `simulating_scenarios`.

**Review by MaGering — APPROVED** [review]
Thanks for building this! :) So far I have just the one [comment](https://github.com/greco-project/pvcompare/pull/221/files#r570214136) I left. I will test in more detail when I run simulations using this feature.

File: pvcompare/outputs.py (hunk 1 above)
Comment (LiMA-programm) [reply pvcompare/outputs.py:269]: yes, you are right. So strange that it somehow worked for me anyways..

Comment (Piranias) [reply pvcompare/outputs.py:269]: yes, you are right. So strange that it worked for me anyway..

**Pull request merged**